database/migrations/2020_05_07_100000_create_refunds_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
             $table->unsignedInteger('amount'); // Amount is in cents.
             $table->string('method');
             $table->string('transaction_number')->nullable();
-            $table->foreignIdFor(app('voucher'))->nullable();
+            $table->unsignedBigInteger('voucher_id')->nullable()->index();
```

Comment:
Necessary to eliminate explicit dependency on Voucher package during real installation.  No loss of functionality, just the loss of DB level FK integrity checks - but no worse than any of the morph relations.